index.css

```diff
@@ -241,6 +318,10 @@ footer p {
     }
 }
 
-#infoContainer {
```

Comment:
This should not be removed, possibly happened due to the merge of master.

Reply:
I update the function on script, maybe it's working fine now, when I did pull the new changes from main, happen some conflicts, so I did some changes to be align with the main branch

Reply:
Awesome, thank you for adjusting the PR. 
Much appreciated!